### ISSUE TITLE
Old `nodejs` installed causes Bokeh error on CI

### ISSUE THREAD

**Issue by jakirkham** [rapidsai/cuxfilter#384]
On [CI]( https://gpuci.gpuopenanalytics.com/blue/organizations/jenkins/rapidsai%2Fgpuci%2Fcuxfilter%2Fprb%2Fcuxfilter-gpu-build/detail/cuxfilter-gpu-build/1321/pipeline ), am seeing the following error about `nodejs` being too old. While it seems [possible to install `nodejs` version `14.0.0`]( https://github.com/rapidsai/cuxfilter/blob/8d3979b0dfd8080d54067a4c0b0e4a9bb9e6dce3/conda/recipes/cuxfilter/meta.yaml#L43 ), CI is picking up an older version of `nodejs` (namely `12.4.0`), which causes this Bokeh error.

```python
/opt/conda/envs/rapids/lib/python3.8/site-packages/bokeh/util/compiler.py in _detect_nodejs()
    410     # if we've reached here, no valid version was found
    411     version = ".".join(map(str, nodejs_min_version))
--> 412     raise RuntimeError('node.js v%s or higher is needed to allow compilation of custom models ' % version +
    413                        '("conda install nodejs" or follow https://nodejs.org/en/download/)')
    414 

RuntimeError: node.js v14.0.0 or higher is needed to allow compilation of custom models ("conda install nodejs" or follow https://nodejs.org/en/download/)
```

**Comment by ajschmidt8** [comment]
@jakirkham, I opened the PRs below to hopefully resolve this.

- https://github.com/rapidsai/cuxfilter/pull/385
- https://github.com/rapidsai/integration/pull/509

**Comment by ajschmidt8** [comment]
Fixed in #385. Closing
